### PR TITLE
Add Qt6 to CI (OS/X)

### PR DESCRIPTION
**Pull request by data-man**

### CODE REVIEW

File: scripts/install-deps.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ main_linux()
 
 main_darwin()
 {
-    brew install freetype fontconfig qt5 harfbuzz boost pkg-config
+    brew install freetype fontconfig qt5 qt6 harfbuzz boost pkg-config
```
Comment (christianparpart) [inline scripts/install-deps.sh:52]: Given the hope that this script is purely for CI and developers, I think including qt5 _and_ qt6 is fine.

**Review by christianparpart — APPROVED** [review]

**Comment by data-man** [comment]
Build error as expected. :)

**Comment by uspasojevic96** [comment]
Build error occurs because brew doesn't auto export Qt variables, you can fix it by doing:

`export Qt5_DIR=$(brew --prefix qt5)/lib/cmake` and `export Qt6_DIR=$(brew --prefix qt6)/lib/cmake`

**Comment by data-man** [comment]
@uspasojevic96  #329

**Comment by christianparpart** [comment]
> @uspasojevic96 #329

I mean, it's easy to merge #329 but then we have a behavior-bug again, i.e. Alt+WheelUp and Alt+WeelDown are all out of the sudden Alt+WheelLeft & Alt+WheelRight.

**Comment by data-man** [comment]
@christianparpart I think this PR is ready. :)

**Comment by christianparpart** [comment]
> @christianparpart I think this PR is ready. :)

Hey, I'll merge right after #329 has a solution. Promised. ^^

**Pull request merged**